### DISCUSSION TITLE
BRD and SCH files?

### DISCUSSION THREAD

**Discussion by jansjobe** [shanteacontrols/OpenDeck#130] [General]
Hi,

Just found this excellent project; exactly what I've been looking for! Fantastic!
So I downloaded the latest Eagle software to view boards andschematics, but I am not able to open the BRD and SCH files I've downloaded. What am I doing wrong?

Also, while I'm at it: is there plans on ART-NET?

And what resolution does DMX currently use, 256 or 127?

Best Regards,
Jan Sjöberg

**Comment by paradajz** [comment]
Hey,

well the schematics are made in fairly old EAGLE version - 7. I actually don't know what's the latest one, I haven't been keeping up with that. You can still download it here:
http://eagle.autodesk.com/eagle/software-versions/1

ART-NET: No idea. Maybe.
DMX is currently controlled through OLA, so full resolution is used (0-255).

**Comment by jansjobe** [comment]
Hi,
Thanks for rapid reply!
I managed to download BRD an SCH as HTML... My bad. Works fine in latest Eagle!
